powerboardcode/powerboard/powerboard.ino

```diff
@@ -88,101 +88,88 @@ void loop()
                     }
                 }
                 break;
-            case RC_POWERBOARD_VACUUMENABLE_DATA_ID:
-                Serial.println("Enable/Disable Vacuum Busses");
-                if (packet.data[0] == 1)
-                {
-                    Serial.println("Enabling Bus:");
-                    Serial.println(0);
-                    digitalWrite(vacuumCtrl[0], HIGH);
-                }
-                else
-                {
-                    Serial.println("Disabling Bus:");
-                    Serial.println(0);
-                    digitalWrite(vacuumCtrl[0], LOW);
-                }
-                break;
         }
     }
 }
 
 void setPins()
 {
-    // sets input/output of pack busses and 12V current sense
-    for (int i = 0 ; i < 4 ; i++)
+    // sets POE and Aux(Spare) to OUTPUT
+    for (int i = 0 ; i < 2 ; i++)
     {
-        pinMode(currentSense12V[i], INPUT);
+        //pinMode(currentSense12V[i], INPUT);
         pinMode(bussesPack[i], OUTPUT);
     }
     // sets input/output of pack current sense
-    for (int i = 0 ; i < 10 ; i++)
+    /*for (int i = 0 ; i < 10 ; i++)
     {
         pinMode(currentSensePack[i], INPUT);
     }
-    // sets output of motor busses
-    for (int i = 0 ; i < 5 ; i++)
+    */
+    // sets motor busses to OUTPUT
+    for (int i = 0 ; i < 7 ; i++)
     {
         pinMode(bussesMotor[i], OUTPUT);
     }
     // sets output of 12V Actuation
-    for (int i = 0 ; i < 3 ; i++)
+    for (int i = 0 ; i < 5 ; i++)
     {
         pinMode(actuation12V[i], OUTPUT);
     }
     // sets output of 12V Logic
-    for (int i = 0 ; i < 7 ; i++)
+    for (int i = 0 ; i < 8 ; i++)
     {
         pinMode(logic12V[i], OUTPUT);
     }
-    pinMode(drivePack, OUTPUT);
-    pinMode(vacuumCtrl[0], OUTPUT);
 }
 
 void setPinStates()
 {
     // turn on 12 volt busses
-    for (int i = 0 ; i < 7 ; i++)
+    for (int i = 0 ; i < 8 ; i++)
```

Comment:
The upper limits of these for loops (and the rest of them like this) should be made into consts, so you don't have to go through and find all of them any time the manifest changes.